packages/redirect-parser/ava.config.js

```diff
@@ -1,6 +1,10 @@
 const config = {
-  files: ['tests/*.js'],
+  files: ['tests/*.ts'],
```

Comment:
add the except helpers thing and move the `tests/helpers/main.js` to typescript as well

Reply:
This is already updated. Github showing the old version of the file.

Reply:
This is already updated. Github showing the old version of the file.